roles/backup/tasks/postgres.yml

```diff
@@ -42,6 +42,7 @@
     namespace: '{{ ansible_operator_meta.namespace }}'
     label_selectors:
       - "{{ postgres_label_selector }}"
+      - app.kubernetes.io/version: '{{ postgres_version }}'
```

Comment:
this is what is breaking the CI, this variable is undefined on backup role

Reply:
ahh, makes sense, thanks.  I just updated that.  Hopefully that passes this next go round.